UVAtlas doc page updated to point to GitHub

walbourn opened the pull request.

walbourn committed:
UVAtlas doc page updated to point to GitHub

PRMerger8 commented:
@walbourn : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edit pass

FWLinks apparently cause loc issues, so we don't use them in the content any longer.

PRMerger7 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
